docs/backend-system/building-backends/08-migrating.md

```diff
@@ -1063,3 +1063,289 @@ backend.start();
 ```
 
 > You can track the progress of the module migration efforts [here](https://github.com/backstage/backstage/issues/19476).
+
+### The Search Plugin
+
+A basic installation of the Search plugin will look as follows:
+
+```ts title="packages/backend/src/index.ts"
+const backend = createBackend();
+
+// Other plugins...
+
+/* highlight-add-start */
+backend.add(import('@backstage/plugin-search-backend/alpha'));
+/* highlight-add-end */
+```
+
+> Note: this will use the Lunr search engine which stores its index in memory
+
+#### Search Engines
+
+The following sections outline how you can add other Search engines
+
+##### Postgres
+
+An installation of the Search plugin using the Postgres search engine will look as follows:
+
+```ts title="packages/backend/src/index.ts"
+const backend = createBackend();
+
+// Other plugins...
+
+/* highlight-add-start */
+backend.add(import('@backstage/plugin-search-backend/alpha'));
+backend.add(import('@backstage/plugin-search-backend-module-pg/alpha'));
+/* highlight-add-end */
+```
+
+##### Elasticsearch
+
+A basic installation of the Search plugin using the Elasticsearch search engine will look as follows:
+
+```ts title="packages/backend/src/index.ts"
+const backend = createBackend();
+
+// Other plugins...
+
+/* highlight-add-start */
+backend.add(import('@backstage/plugin-search-backend/alpha'));
+backend.add(
+  import('@backstage/plugin-search-backend-module-elasticsearch/alpha'),
+);
+/* highlight-add-end */
+```
+
+#### Search Collators
+
+The following sections outline how you add the Catalog and TechDocs search collators
```

Comment:
```suggestion
The following sections outline how you add search collators (input sources for the search indexing process).
```

just to make it easier to expand in the future without missing updating the text here too